From 3020891009a6024b46bc779568644ce16605bda9 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Mon, 20 Jul 2015 14:10:36 +0000
Subject: [PATCH] abuild: add install_if for -doc packages

This lets users 'apk add docs' to always pull in -doc packages.
---
 abuild.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/abuild.in b/abuild.in
index e76d873..41822af 100644
--- a/abuild.in
+++ b/abuild.in
@@ -1391,6 +1391,7 @@ default_doc() {
 	depends="$depends_doc"
 	pkgdesc="$pkgdesc (documentation)"
 	arch=${arch_doc:-"noarch"}
+	install_if="docs $pkgname=$pkgver-r$pkgrel"
 
 	local i
 	for i in doc man info html sgml licenses gtk-doc ri help; do
-- 
2.4.6

